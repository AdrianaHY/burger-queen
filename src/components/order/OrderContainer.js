import React, { Component } from 'react';

import trash from '../../assets/paper-bin.svg'

import './OrderContainer.scss';

class OrderContainer extends Component {
	render() {
		const { order, deleteItem } = this.props;
		return (
			<div className="order-container">
				<div className="order">
					{order.map((item) => (
						<div className="order-item" key={item}>
							<p className="order-text">{item}</p>
							<button onClick={() =>deleteItem(item)}>
								<img  src={trash} alt="trash" className="trash" />
							</button>
						</div>
					))}
				</div>
				<button className="button-kitchen">Send to kitchen</button>
			</div>
		)
	}
}

export default OrderContainer;