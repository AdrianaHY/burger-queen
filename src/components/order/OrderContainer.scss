@import '../../styles/_variables.scss';

.order-container {
	text-align: center;
	font-size: $fontSizeMedium;
	font-family: $fontFamily;

	.order {
		background: $lightYellow;
		margin: $paddingHuge $paddingHuge*3;
	}

	.order-item {
		display: flex;
		align-items: center;
		justify-content: space-around;
		padding: $paddingTiny;
		height: 44px;
		border-bottom: 1px solid $orange;
	}

	.trash {
		height: 30px;
	}

	.button-kitchen {
		border: 2px solid $purple;
		color: $purple;
		background: transparent;
		padding: $paddingMedium;
		font-size: $fontSizeHuge;
		font-family: $fontFamily;
		width: 30%;
		border-radius: 5%;
	}
}