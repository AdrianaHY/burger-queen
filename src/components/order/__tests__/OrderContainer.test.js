import React from 'react';
import {Â shallow } from 'enzyme';
import OrderContainer from '../OrderContainer';

describe('components | order | Order Container', () => {
	it('Renders the component', () => {
		const props = {
			order: [
				'soda',
				'sandwich'
			],
			deleteItem: jest.fn()
		};

		const component = shallow(
			<OrderContainer {...props} />
		);

		expect(component).toMatchSnapshot();
	})
})