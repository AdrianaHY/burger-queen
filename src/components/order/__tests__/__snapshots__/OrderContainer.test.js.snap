// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | order | Order Container Renders the component 1`] = `ShallowWrapper {}`;
